graphein/ml/datasets/pdb_data.py

```diff
@@ -146,6 +163,9 @@ def download_metadata(self):
         self._download_entry_metadata()
         self._download_exp_type()
         self._download_pdb_availability()
+        self._download_pdb_chain_cath_uniprot_map()
```

Comment:
Are these slow to download? If they are, can we make them only run if we are using `labels`? If not, don't worry about it.

Reply:
Thought about that, but they are quite quick to download